metrics-operator/main.go

```diff
@@ -182,6 +182,12 @@ func main() {
 
 	if env.EnableKeptnAnalysis {
 
+		// OTel Setup
+		metrics, err := analysiscontroller.SetupMetric()
+		if err != nil {
```

Comment:
shouldn't we return here? If we have error here, which can come from `Register()`, and afterwards we proceed, can we potentially bump to error in this place ? https://github.com/keptn/lifecycle-toolkit/pull/2046/files#diff-8f9bed941d90a04db82a926c46ff3e9c07ff65ef32293f3461f2492a68bb09eaR160

Reply:
I think we should have a more fine-grained error check here (like in the tests) where we allow it to continue iff it is an "already registered" error. 
The line you linked won't fail because we failed here. It fails if there are problems setting the value on the metric. If we fail here, the metrics are created but never exported.

Reply:
nvm, I looked into Prometheus and they don't have any well-defined errors that export. Having an error string comparison doesn't make sense. Let's return here 👍